### PR TITLE
fix(labelWithAdjustments): add override for attributedText.

### PR DESCRIPTION
**Pull request by ivan-fueled**

### CODE REVIEW

**Review by andy-fueled — APPROVED** [review]

File: FueledUtils/LabelWithTitleAdjustment.swift [hunk 1]
```diff
@@ -24,6 +24,16 @@ open class LabelWithTitleAdjustment: UILabel {
 		}
 	}
 
+	override open var attributedText: NSAttributedString? {
+		get {
+			return super.attributedText
```
Comment (ghost) [inline FueledUtils/LabelWithTitleAdjustment.swift:29]: @ivan-fueled why override the getter if you only call super's implementation?

File: FueledUtils/LabelWithTitleAdjustment.swift [hunk 2]
```diff
@@ -36,7 +46,7 @@ open class LabelWithTitleAdjustment: UILabel {
 
 	private func setAdjustedAttributedText(_ text: NSAttributedString?) {
 		guard let text = text else {
-			self.attributedText = nil
+			super.attributedText = nil
```
Comment (ghost) [inline FueledUtils/LabelWithTitleAdjustment.swift:49]: I'd add a comment that this is done in order to avoid recursion. It's not immediately obvious.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@ivan-fueled I suggest that you consider the comments but because they are not essential, I'm merging this to make use of this on our project asap.

File: FueledUtils/LabelWithTitleAdjustment.swift (hunk 2 above)
Comment (ivan-fueled) [reply FueledUtils/LabelWithTitleAdjustment.swift:49]: 👍

**Pull request merged**